python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py

```diff
@@ -1,278 +1,463 @@
+# +-------+----------------+
+# | Index | Input variable |
+# +=======+================+
+# | 0     | Strain         |
+# +-------+----------------+
+#
+# +-------+-----------+
+# | Index | Parameter |
+# +=======+===========+
+# | 0     | R         |
+# +-------+-----------+
+# | 1     | C         |
+# +-------+-----------+
+# | 3     | Gamma     |
+# +-------+-----------+
+#
+# **Table 2.** Indices and names of the inputs and parameters of the parametric model.
+#
 
 # %%
+# The following statement create the calibrated function from the model.
+# The calibrated parameters `R`, `C`, `Gamma` are at indices 1, 2, 3 in
+# the inputs arguments of the model.
 calibratedIndices = [1, 2, 3]
-mycf = ot.ParametricFunction(g, calibratedIndices, thetaPrior)
+mycf = ot.ParametricFunction(cm.model, calibratedIndices, thetaPrior)
 
 # %%
-# Calibration with linear least squares
-# -------------------------------------
+# Then we plot the model and compare it to the observations.
+graph = ot.Graph("Model before calibration", "Strain", "Stress (MPa)", True)
+# Plot the model
+curve = mycf.draw(cm.strainMin, cm.strainMax, 50).getDrawable(
+    0
+)
+curve.setLegend("Model before calibration")
+curve.setLineStyle(ot.ResourceMap.GetAsString("CalibrationResult-ObservationLineStyle"))
+graph.add(curve)
+# Plot the noisy observations
+cloud = ot.Cloud(observedStrain, observedStress)
+cloud.setLegend("Observations")
+cloud.setPointStyle(
+    ot.ResourceMap.GetAsString("CalibrationResult-ObservationPointStyle")
+)
+graph.add(cloud)
+graph.setColors(ot.Drawable.BuildDefaultPalette(2))
+graph.setLegendPosition("topleft")
+view = viewer.View(graph)
 
 # %%
-# The `LinearLeastSquaresCalibration` class performs the linear least squares calibration by linearizing the model in the neighbourhood of the reference point.
+# We see that the observations are relatively noisy, but that the
+# trend is clear: this shows that it may be possible to fit the model.
+# At this point, we have a data set that we can use for calibration
+# and a model to calibrate.
+
 
 # %%
+# Calibration with linear least squares
+# -------------------------------------
+# The :class:`~openturns.LinearLeastSquaresCalibration` class performs the
+# linear least squares
+# calibration by linearizing the model in the neighbourhood of the reference point.
 algo = ot.LinearLeastSquaresCalibration(
     mycf, observedStrain, observedStress, thetaPrior, "SVD"
 )
 
 # %%
-# The `run` method computes the solution of the problem.
-
-# %%
+# The :meth:`~openturns.LinearLeastSquaresCalibration.run` method computes
+# the solution of the problem.
 algo.run()
-
-# %%
 calibrationResult = algo.getResult()
 
 # %%
 # Analysis of the results
 # -----------------------
-
-# %%
-# The `getParameterMAP` method returns the maximum of the posterior distribution of :math:`\theta`.
-
-# %%
+# The :meth:`~openturns.CalibrationResult.getParameterMAP` method
+# returns the maximum of the posterior distribution of :math:`\theta`.
```

Comment:
```suggestion
# returns the maximum of the posterior density of :math:`\theta`.
```
It is likely that this correction can be made at other places.

Reply:
Accurate. Ok, fixed.